### ISSUE TITLE
How to do e2e_qp with multi-GPU?

### ISSUE THREAD

**Issue by mxjmtxrm** [OpenGVLab/EfficientQAT#14]
As title. I set `CUDA_VISIBLE_DEVICES=0,1,2,3` before `python main_e2e_qp.py --xxxxx`. I found that only GPU0 was used during training. Is there any parameters I missed?

**Comment by ChenMnZ** [comment]
Current code only support for the single-GPU training.

**Comment by brisker** [comment]
@ChenMnZ 
The LLM-QAT paper use the model-output-logits of quantized model and fp model to do the distill loss.
In e2e qp in efficient-QAT, **is the loss the same as LLM-QAT?** while the main diffenrence is that, llm-qat train all the weights, but efficient-qat e2e only train quanti scales?
